### PR TITLE
Began to de-emphasize Lambda (Do Not Merge)

### PR DESCRIPTION
**Pull request by cppwfs**
- Added text stating XD doesn't pick a deployment type its up to the user to select their strategy, rather XD gives you the ability to build what you need.
- Removed Lambda Graph
- Condensed some text.
- Removed mentions of backpressure and Spring Reactor.

### CODE REVIEW

File: spring-xd-deployment.tex [hunk 1]
```diff
@@ -1,30 +1,18 @@
 \section{Deployment}
 This section describes deployment architectures that are possible with Spring XD.
-Note that this is not an exhaustive list of all deployment types. See
-section~\ref{sec:Use Cases} for descriptions of real world deployments.
+While XD does not subscribe to a specific deployment model (Lambda,
```
Comment (pperalta) [inline spring-xd-deployment.tex]: Should we eliminate the middle sentences so this reads:

> This section describes deployment architectures that are possible with Spring XD. Note that this is not an exhaustive list of all deployment types, see section 7 for descriptions of real world deployments.

File: spring-xd-deployment.tex [hunk 2]
```diff
@@ -1,30 +1,18 @@
...
-section~\ref{sec:Use Cases} for descriptions of real world deployments.
+While XD does not subscribe to a specific deployment model (Lambda, 
+Reactive or other), it does allow for a user to architect the deployment type 
+that meets their needs.  Note that this is not an exhaustive list of all 
+deployment types, see section~\ref{sec:Use Cases} for descriptions of real 
+world deployments.
 
 \subsection{Lambda Architecture}
 
-The Lambda Architecture, introduced by Nathan Marz \cite{lambda-architecture-paper}
```
Comment (pperalta) [inline spring-xd-deployment.tex:8]: Can we keep the original opening paragraph? Then add this to it:

> Spring XD predates the Lambda Architecture and was not specifically created for it. However, Spring XD's streaming and batching support lines up well with the various components of the architecture.

File: spring-xd-deployment.tex [hunk 3]
```diff
@@ -1,30 +1,18 @@
...
-and shown in figure~\ref{fig:lambda}
-is a generic, scalable and fault tolerant data  processing architecture. It 
+One of the architectures a user can implement using XD is the
+Lambda Architecture, introduced by Nathan Marz \cite{lambda-architecture-paper}. 
+Is a generic, scalable and fault tolerant data  processing architecture that 
 attempts to provide a comprehensive solution to the problem of processing an 
-extremely large set of data.
-
-The Lambda Architecture has the following components:
```
Comment (pperalta) [inline spring-xd-deployment.tex:14]: I'm split between keeping this section and removing it as proposed in the PR. @mbogoevici any opinions?

File: spring-xd-deployment.tex [hunk 4]
```diff
@@ -37,13 +25,6 @@ \subsection{Lambda Architecture}
...
 
-\begin{figure}[ht]
-\centering
-\epsfig{file=lambda.eps, height=2in, width=3in}
-\caption{Lambda Architecture.}
-\label{fig:lambda}
-\end{figure}
-
 In the following subsections we will examine how the different components
 of the Lambda architecture can be implemented with Spring XD.
```
Comment (pperalta) [inline spring-xd-deployment.tex]: I added some thoughts to the DEBS feedback document for this section:

https://docs.google.com/document/d/10wtQ_Gqcw_y0GxCZVe1ARJYlQ3M_ab6ZipUBFdr2GSY/edit#bookmark=id.ivfohl6w35zq

File: spring-xd-deployment.tex (hunk 2 above)
Comment (mbogoevici) [inline spring-xd-deployment.tex:8]: I would remove this section header altogether and simply have three dedicated subsections for streaming, batch and stream+batch.

File: spring-xd-deployment.tex (hunk 3 above)
Comment (mbogoevici) [inline spring-xd-deployment.tex:14]: I would remove the Lambda architecture as a top level section. We could add a paragraph about the existence of stream and batch support in Spring XD makes lambda possible, but no more than that.

**Comment by mbogoevici** [comment]
Merged w/https://github.com/pperalta/spring-xd-debs-2015/commit/09865c476f6fb86232652b58f51274cf0e723291

**Pull request closed without merging**